python/cudf_polars/cudf_polars/experimental/parallel.py

```diff
@@ -184,28 +178,15 @@ def evaluate_streaming(ir: IR, config_options: ConfigOptions) -> DataFrame:
 def _(
     ir: IR, partition_info: MutableMapping[IR, PartitionInfo]
 ) -> MutableMapping[Any, Any]:
-    # Single-partition default behavior.
-    # This is used by `generate_ir_tasks` for all unregistered IR sub-types.
-    if partition_info[ir].count > 1:
-        raise NotImplementedError(
-            f"Failed to generate multiple output tasks for {ir}."
-        )  # pragma: no cover
-
-    child_names = []
-    for child in ir.children:
-        child_names.append(get_key_name(child))
-        if partition_info[child].count > 1:
-            raise NotImplementedError(
-                f"Failed to generate tasks for {ir} with child {child}."
-            )  # pragma: no cover
-
-    key_name = get_key_name(ir)
```

Comment:
Now that multi-partition support is 100% "handled" at the IR-lowering level (i.e. in `lower_ir_node`), the default behavior of `generate_ir_tasks` can be the same as `_generate_ir_tasks_pwise`. Therefore, this PR removes `_generate_ir_tasks_pwise`, and copies the same logic here.